spec/models/events/review_comment_spec.rb

```diff
@@ -73,27 +84,33 @@
     end
 
     describe '#created' do
+      before { subject.payload['action'] = 'created' }
+
       it 'sets body' do
         expect {
-          subject.send(:created)
+          subject.resolve
         }.to change { subject.reload.body }.from(nil).to(payload['comment']['body'])
       end
     end
 
     describe '#edited' do
+      before { subject.payload['action'] = 'edited' }
```

Comment:
This is a style question.
Would it make sense to define a method

```
def set_payload_action_to(action)
   subject.payload['action'] = action
end
```

at the beginning of the spec file and to change

```
before { subject.payload['action'] = 'edited' }
```

to

```
before { set_payload_action_to 'edited' }
```

?

Reply:
Yes! That would be great. Thanks!